diff --git a/node_modules/pretender/dist/pretender.js b/node_modules/pretender/dist/pretender.js
index 729003f..2e73bf6 100644
--- a/node_modules/pretender/dist/pretender.js
+++ b/node_modules/pretender/dist/pretender.js
@@ -46,9 +46,15 @@ var Pretender = (function (FakeXMLHttpRequest, FakeFetch, RouteRecognizer) {
    * }
    */
   function parseURL(url) {
-      // TODO: something for when document isn't present... #yolo
-      var anchor = document.createElement('a');
-      anchor.href = url;
+      var anchor;
+      try {
+          anchor = document.createElement('a');
+          anchor.href = url;
+      } catch {
+          var URL = require('url-parse');
+          anchor = new URL(url);
+      }
+    
       if (!anchor.host) {
           // eslint-disable-next-line no-self-assign
           anchor.href = anchor.href; // IE: load the host and protocol
